packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -600,6 +605,10 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
     return defaultComputeDistanceToFirstActualBaseline(baseline);
   }
 
+  // Hints sent to console when the container overflows.
+  // Only set in debug mode.
+  String _debugOverflowHints;
```

Comment:
there's not much point caching this. The cost of building that string is barely anything and debug-mode only, whereas the memory cost of the String field is being paid in every Flex of every release-mode app.